package.json

```diff
@@ -7,44 +7,52 @@
   "dependencies": {
     "@material-ui/core": "^4.9.1",
     "babel-plugin-styled-components": "^1.10.6",
+    "core-js": "^3.6.5",
     "firebase": "^7.8.1",
+    "fsevents": "^2.1.2",
     "gatsby": "^2.19.12",
     "gatsby-background-image": "^0.9.14",
     "gatsby-image": "^2.2.40",
     "gatsby-plugin-catch-links": "^2.1.25",
     "gatsby-plugin-firebase": "^0.1.7",
     "gatsby-plugin-google-analytics": "^2.1.36",
-    "gatsby-plugin-manifest": "^2.2.41",
+    "gatsby-plugin-manifest": "^2.3.3",
     "gatsby-plugin-offline": "^3.0.34",
     "gatsby-plugin-react-helmet": "^3.1.22",
     "gatsby-plugin-sass": "^2.1.28",
-    "gatsby-plugin-sharp": "^2.4.5",
+    "gatsby-plugin-sharp": "^2.5.4",
     "gatsby-plugin-styled-components": "^3.1.19",
+    "gatsby-plugin-web-font-loader": "^1.0.4",
     "gatsby-remark-images": "^3.1.44",
     "gatsby-remark-relative-images": "^0.2.2",
     "gatsby-source-filesystem": "^2.1.48",
     "gatsby-transformer-remark": "^2.6.50",
-    "gatsby-transformer-sharp": "^2.3.14",
+    "gatsby-transformer-sharp": "^2.4.4",
     "intersection-observer": "^0.7.0",
+    "jquery": "^3.5.0",
```

Comment:
Was this added just because it is a dependency of another package?
react and react-dom can stay as regular dependencies but it doesn't really matter.

Reply:
The jQuery must just be another dependency since I didn't add that on purpose.
I was getting errors when running "gatsby develop" that it wasn't finding the proper versions of react and react-dom. So, I installed new versions. If they need to be moved I can try that and see if I have any issues.